### PR TITLE
Ascii renderer test enhancement

### PR DESCRIPTION
**Pull request by corlaez**
* ASCII render test now uses ascii instead of unicode
* Previous test failed in Windows, now it should work in any platform
* Previous test only checked the length, added a snapshot of the expected output

### CODE REVIEW

File: src/test/java/ch/astorm/jchess/util/ASCIIPositionRendererTest.java [hunk 1]
```diff
@@ -9,17 +9,66 @@
 import org.junit.jupiter.api.Test;
 
 public class ASCIIPositionRendererTest {
+    private final String separator = System.lineSeparator();
+    private final String expectedOutput = "" +
```
Comment (ctabin) [inline src/test/java/ch/astorm/jchess/util/ASCIIPositionRendererTest.java:13]: Can you please move the content in a file and load it with `ASCIIPositionRendererTest.class.getResourceAsStream(...)`.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
 /nbproject/
-/target/
\ No newline at end of file
+/target/
+.idea/
+jchess.iml
```
Comment (ctabin) [inline .gitignore:4]: I don't see why `jchess.iml` should be ignored :thinking:

```suggestion
/.idea/
```

**Review by ctabin — CHANGES_REQUESTED** [review]
Thanks for your contribution ! I left some comments.

File: .gitignore (hunk 2 above)
Comment (corlaez) [reply .gitignore]: I usually do that but I don't even need it here. Removed.

File: src/test/java/ch/astorm/jchess/util/ASCIIPositionRendererTest.java (hunk 1 above)
Comment (corlaez) [reply src/test/java/ch/astorm/jchess/util/ASCIIPositionRendererTest.java]: Moved

**Pull request merged**